WebRoot/js/ajax/config/msgs/AjxMsg_vi.properties

```diff
@@ -0,0 +1,157 @@
+# 
```

Comment:
thanks, probably I will look at removing english version file as well if not required